packages/flutter/lib/material.dart

```diff
@@ -49,6 +49,7 @@ export 'src/material/circle_avatar.dart';
 export 'src/material/color_scheme.dart';
 export 'src/material/colors.dart';
 export 'src/material/constants.dart';
+export 'src/material/curves.dart';
```

Comment:
Can we do this one in a separate PR to make a potential revert easier? This one is a little more risky and may break things (see animations package).

Reply:
ack - will update it tomorrow morning

Reply:
Done